packages/docusaurus-plugin-content-docs/src/routes.ts

```diff
@@ -235,6 +255,10 @@ export async function buildAllRoutes(
       exact: false,
       component: param.options.docsRootComponent,
       routes: subRoutes,
+      lastModified: compositeLastModified(
+        subRoutes,
+        (subRoute) => subRoute.lastModified,
+      ),
```

Comment:
"parent routes" are wrapping "leaf routes" and only "leaf routes" are creating sitemap entries. This lastModified attribute here will never be used in practice.